### PR TITLE
Project-survey, Marielle Ohlsson

### PR DESCRIPTION
**Pull request by Marielle8**
Link to my deployed project: https://book-survey.netlify.app/

My reflections on how this week's project turned out: I'm very proud over the result since I managed to understand everything I did with the code during the project.

Things I'd like to have clarified or explained in more detail: I aimed for the blue level for this project.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
+    };
+    if (numberOfBook === "") {
+      return false
+    };
+    if (recommendation === "") {
+      return false
+    };
+    return true
+  };
+
```
Comment (petercreations) [inline code/src/App.js:51]: Well done on the form validation! 👍

File: code/src/components/RadioButtonsRecommends.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const RadioButtonsRecommends = (props) => {
+    return (
+        <div className="radio-container">
+            <p tabIndex="0">Would you recommend the book to someone else?</p>
+            <div className="radio-buttons-container">
+                {recommendationAnswer.map((answer) => (
+                    <label
+                        tabindex="0"
+                        htmlFor="books"
+                        key={answer}
```
Comment (petercreations) [inline code/src/components/RadioButtonsRecommends.js:14]: Well done remembering a key and good use of .map !

File: code/src/components/Summary.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <div className="summary-container">
+            <section className="summary">
+                <div className="summary-header-container">
+                    <h2 tabIndex="0">Hi {props.name}!</h2>
+                </div>
+                <p tabIndex="0">The latest book you have read was: {props.bookname}</p>
+                <p tabIndex="0">After you had finished {props.bookname}, you felt {props.feeling}!</p>
+                <p tabIndex="0">This is how many times you have read the book: {props.numberOfBook}</p>
+                <p tabIndex="0">If someone would ask you if they should read {props.bookname}, your answer will be: {props.recommendation}!</p>
+                <p tabIndex="0">Thank you for your time and have a great day!</p>
```
Comment (petercreations) [inline code/src/components/Summary.js:14]: Excellent use of tabIndex through the app for accessibility 🎉

File: code/src/components/Summary.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+                <p>Photo by <a href="https://unsplash.com/@chrislawton?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Chris Lawton</a> on <a href="/s/photos/books?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+                </p>
+            </section>
+        </div>
+    );
+};
+
+export default Summary;
+
```
Comment (petercreations) [inline code/src/components/Summary.js:23]: Not sure if this is convention but you could add an empty line below each export default like this for readability ?

File: code/src/index.css [hunk 5]
```diff
@@ -11,3 +15,190 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.form-container {
+  width: 100vw;
+  height: 100vh;
```
Comment (petercreations) [inline code/src/index.css:21]: Very nice styling !

File: README.md [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
+
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://book-survey.netlify.app/
```
Comment (petercreations) [inline README.md:15]: This is also an option if you want to use markdown to hide the link: [I'm an inline-style link](https://www.google.com)

**Review by petercreations — APPROVED** [review]
Very nice idea and project all round ! 😀  I believe you have achieved all the Blue requirements.  

For the Summar.js 

If you wanted to give the user an option to reload thepagee with a button click you could add this restart function and this button on the summary page:

```
   const restart = () => {
        document.location.href = '';
      };
```


    `<button className="restart-button" onClick={restart} type="button">Restart</button>
`

**Review by KarolCislowski — APPROVED** [review]
Great job!
Maybe for the next step, you will try to add some more exotic types of inputs? 🍹

**Pull request closed without merging**